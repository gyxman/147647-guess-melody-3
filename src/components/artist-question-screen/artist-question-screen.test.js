import React from "react";
import renderer from "react-test-renderer";
import {ArtistQuestionScreen} from "./artist-question-screen.jsx";

it(`Если приложение загрузилось, то компонент ArtistQuestionScreen отрисовался`, () => {
  const tree = renderer
    .create(<ArtistQuestionScreen
      question={{song: {src: ``, artist: ``}, answers: []}}
      onAnswer={() => {}}
      renderPlayer={() => {}}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
