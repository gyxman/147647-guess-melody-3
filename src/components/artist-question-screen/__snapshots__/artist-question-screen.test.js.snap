// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Если приложение загрузилось, то компонент ArtistQuestionScreen отрисовался 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    />
  </div>
  <form
    className="game__artist"
  />
</section>
`;
